src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -293,7 +293,7 @@ export default defineComponent({
           copyToClipboard(err)
         })
         if (process.env.IS_ELECTRON && this.backendFallback && this.backendPreference === 'invidious') {
-          showToast(this.$t('Falling back to local API'))
+          showToast(this.$t('Falling back to the local API'))
```

Comment:
In all our documentation it is referred as **L**ocal API, maybe best to keep it that way?

Reply:
I updated the English translations to be `Local API`

Reply:
The IV version is `Falling back to Invidious API`
I don't see a reason to use `the local API` as key, should be `Falling back to Local API` (not just translation value, but also the key)

Reply:
Updated the translation keys

Reply:
But the key in code still unchanged

Reply:
Oops, i just did a replace all, i gues it ignored js files. Will fix no2